website/project/metadata/asist-hypothesis-capability-registration.json

```diff
@@ -0,0 +1,32 @@
+{
+   "name":"ASIST Hypothesis/Capability Registration",
+   "version":1,
+   "config":{
+      "hasFiles":true
+   },
+   "description":"This template was created by researchers from the qualitative community for preregistration of qualitative research",
+   "pages":[
+      {
+         "id":"page1",
+         "title":"Research Aims",
+         "questions":[
+            {
+               "qid":"q1",
+               "type":"string",
+               "format":"text",
+               "title":"Study registration link",
+               "description":"Please specify the overall purposes, objectives or aims of the research.\n\nIn addition, please reflect on whether your aim is different across different domains (e.g. knowledge generation, policy development, community resourcing). If so, specify your aim for each domain that is relevant for your study.",
```

Comment:
Where did this text come from? The spreadsheet I have looks more like this:

<img width="638" alt="Screen Shot 2020-10-22 at 4 03 05 PM" src="https://user-images.githubusercontent.com/6599111/96924584-d95c3400-1480-11eb-9e35-cefc3878d31d.png">